assets/message-bus.js

```diff
@@ -490,6 +490,9 @@
       }
 
       if (typeof lastId !== "number") {
+        if (lastId !== null && typeof lastId !== "undefined") {
+          console.log("lastId has type " + typeof lastId + " but a number was expected.  Setting lastId = -1.")
```

Comment:
I am ok with warning against misuse, but `console` many not be defined, we need to check for it.

Reply:
Maybe simpler here and more consistent is just to throw if lastId is provided and is invalid.

Reply:
I thought about `throw` but I was worried that would be a breaking change.

Reply:
Yeah much prefer to do a breaking change here, hiding this in the console is risky, people will not catch it in CI and so on.

Reply:
Great, I wasn't sure you'd want a breaking change for this but it is much cleaner :+1: